security/firely-auth/firely-auth-sso.rst

```diff
@@ -32,6 +32,11 @@ Multiple configuration parts are necessary to enable SSO in Firely Auth:
     These claims will be copied from the ID token after a successful login and stored permanently. Each claim is updated automatically after each login with local changes being overwritten.
     It is possible to assign a new name to a claim using the ``CopyAs`` setting.
 
+#. Configure security groups
+
+    Based on the ``AutoProvisionFromSecurityGroup`` setting it is possible to restrict the sign-up of users based on security groups defined in the SSO provider. The attribution of a user account to a one or more security group needs to be exposed via the ``groups`` claim.
```

Comment:
Can you add documentation on this setting in this page as well?

https://docs.fire.ly/projects/Firely-Server/en/latest/security/firely-auth/firely-auth-settings.html#external-identity-providers